src/pages/debrief/cat-c/debrief.cat-c.page.ts

```diff
@@ -32,8 +32,8 @@ import { FaultCountProvider } from '../../../providers/fault-count/fault-count';
 import { getTestCategory } from '../../../modules/tests/category/category.reducer';
 import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
 import { FaultSummaryProvider } from '../../../providers/fault-summary/fault-summary';
-import { getCandidate } from '../../../modules/tests/journal-data/candidate/candidate.reducer';
-import { getUntitledCandidateName } from '../../../modules/tests/journal-data/candidate/candidate.selector';
+import { getCandidate } from '../../../modules/tests/journal-data/cat-b/candidate/candidate.reducer';
```

Comment:
should cat-b be cat-c

Reply:
This we should fix this later, will add a TODO

Reply:
I actually changed it to cat-c